src/gateway/game.gateway.ts

```
@@ -0,0 +1,105 @@
+    if (game.gameStatus === GameStatus.ENDED) {
+      this.server.to(game._id.toString()).emit('gameEnded', game);
+    }
+  }
+
+  /**
+   * End game manually (abort/timeout/etc)
+   */
+  @SubscribeMessage('endGame')
+  async handleEndGame(
+    @MessageBody() data: { gameId: string; reason: string; winner: string },
+    @ConnectedSocket() client: Socket,
```

ESLint

[error] 'client' is defined but never used.

(@typescript-eslint/no-unused-vars)